torchgeo/models/fcsiam.py

```diff
@@ -134,35 +134,8 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         """Initialize a new FCSiamConc model.
 
         Args:
-            encoder_name: Name of the classification model that will
```

Comment:
Only change in this PR that might be controversial. I think it's better to just point users to the upstream docs and say "ditto".